### PR TITLE
Create Workshop-request-forms-and-more.md

### PR DESCRIPTION
**Pull request by sheraaronhurt**
Please feel free to make any changes :)

### CODE REVIEW

**Review by kariljordan — APPROVED** [review]
I made a few minor edits and changed the file name.

**Comment by kariljordan** [comment]
This blog post needs to be reviewed and merged by @tracykteal. I didn't want to omit @sheraaronhurt's voice from the post, but am not sure if the tone of the first paragraph is too informal. Thoughts @tracykteal?

@sheraaronhurt I am so proud of this project and this post!

**Comment by sheraaronhurt** [comment]
@kariljordan thanks so much! @tracykteal if it the intro needs to be toned down a bit I understand :) I am just excited and wanted the community to feel it too :)

File: _posts/2019/09/Workshop-request-forms-and-more.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+time: "09:00:00"
+tags: ["Website", "Workshops"]
+---
+
+I am sooooooooooooooooo excited! I get to be the first one to tell you that the update to the [Workshop Request Forms](https://amy.carpentries.org/forms/workshop/) and the [Workshop section](https://carpentries.org/workshops/) of The Carpentries website is COMPLETE! Wooohoooo!!!!!
+
+## Updated Workshop Request Forms
+
+With the new changes, workshop organisers no longer have to complete long forms with fields that are not applicable to them. We have made it so that you provide us with only the information needed to process your workshop. We have separated the [Workshop Request Forms](https://amy.carpentries.org/forms/workshop/) based on the different kinds of workshops that your organise: 
+
```
Comment (tracykteal) [inline _posts/2019/09/Workshop-request-forms-and-more.md]: that your organise -> that you organise

File: _posts/2019/09/Workshop-request-forms-and-more.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+*  _Workshop Request Form_ - Use when you are ready to request a centrally organised workshop and you have all of the information needed (date, location, curriculum, etc.)  to begin the process
+*  _Workshop Inquiry Form_ - Use when you  want to host a centrally organised workshop but do not have all of the information or would like to speak to someone to discuss options
+* _Self-Organised Notification Form_ - Use when you have created the workshop website page using our [template](https://github.com/carpentries/workshop-template), would like to get survey links and to have the workshop listed on [The Carpentries website](https://carpentries.org/).
```
Comment (tracykteal) [inline _posts/2019/09/Workshop-request-forms-and-more.md]: -> Self-Organised Notification Form - For Certified Instructors and Member Organisations to use to register a self-organised workshop. Submit your workshop website from our [template]((https://github.com/carpentries/workshop-template) to share the information about your workshop and get survey links.

File: _posts/2019/09/Workshop-request-forms-and-more.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+You now only need to bookmark [one link](https://amy.carpentries.org/forms/workshop/) that will direct you to the form that matches your request. 
+
+## Updated Workshop Webpage
+Finding information regarding workshops is now easily accessible. How to organise a Carpentries workshop, the cost of a workshop, the requirements of a workshop and description of The Carpentries lessons can be found on the [workshop section](https://carpentries.org/workshops/) of [The Carpentries webpage](https://carpentries.org/)
```
Comment (tracykteal) [inline _posts/2019/09/Workshop-request-forms-and-more.md]: accessible - how to organise ...

File: _posts/2019/09/Workshop-request-forms-and-more.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+* _Self-Organised Notification Form_ - Use when you have created the workshop website page using our [template](https://github.com/carpentries/workshop-template), would like to get survey links and to have the workshop listed on [The Carpentries website](https://carpentries.org/).
+
+You now only need to bookmark [one link](https://amy.carpentries.org/forms/workshop/) that will direct you to the form that matches your request. 
+
+## Updated Workshop Webpage
+Finding information regarding workshops is now easily accessible. How to organise a Carpentries workshop, the cost of a workshop, the requirements of a workshop and description of The Carpentries lessons can be found on the [workshop section](https://carpentries.org/workshops/) of [The Carpentries webpage](https://carpentries.org/) 
+
+## Tracking non-standard Workshops
+The Carpentries is excited to let our community members know that we are piloting tracking non-standard workshops. Non-standard workshops can only be self-organized and includes any of the following:
+* taught only one day or over a longer period of time than two days
```
Comment (tracykteal) [inline _posts/2019/09/Workshop-request-forms-and-more.md]: organized -> organised

File: _posts/2019/09/Workshop-request-forms-and-more.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+* teaches individual modules 
+* Mix &  Match Curriculums - Combining modules from any lesson programs ([DC](https://datacarpentry.org/lessons/), [LC](https://librarycarpentry.org/lessons/), [SWC](https://software-carpentry.org/lessons/)) to teach one workshop
+
+As with the standard workshops, there will still be requirements in order for this to be called a Carpentries workshop. 
+* Have at least 1 [certified instructor](https://carpentries.org/instructors/) 
+* Must use The Carpentries surveys to get evaluations
+* Abide by our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
+
```
Comment (tracykteal) [inline _posts/2019/09/Workshop-request-forms-and-more.md]: * Use Carpentries materials

File: _posts/2019/09/Workshop-request-forms-and-more.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Now that we are piloting tracking non-standard workshops, we will accept instructors registering any upcoming workshops. Unfortunately, we do not have the capacity to track previous workshops. This is a pilot, so we’ll be able to better track and evaluate these workshop formats.  
+
+## THANK YOU!
+These changes could not have been possible without the voice of the community. I would like to thank all of the community members who proofread, shared their feedback or offered suggestions. It is our goal to make our resources as accessible as possible. With your help, we have been able to do just that!
```
Comment (tracykteal) [inline _posts/2019/09/Workshop-request-forms-and-more.md]: With your help ... -> With your help, we are working towards this goal!

**Pull request merged**